benchmarks/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "hypercore-logs-benchmark",
```

Comment:
I would prefer to have single package.json here, maybe we can do this:
- install clinic and autocannon as dev dependencies on root package.json
- move start script as benchmark on root package.json `benchmark: clinic doctor --on-port 'autocannon localhost:$PORT' -- node ./tests/$ID.js`
then people can checkout between different tags and perform benchmark tests

Reply:
The dedicated package.json allows to install and test previous versions that don't have this benchmarking system

Reply:
alright it makes sense then